build/release.sh

```diff
@@ -3,6 +3,9 @@ set -e # Any subsequent(*) commands which fail will cause the shell script to ex
 
 SBT_CMD="sbt"
 
+SBT_2_12="sbt -Dspark.include=true -Dscala.version=$(sed -n '5p' < .travis.yml | cut -c5-12)"
```

Comment:
Sorry for taking so long to review. I'm not comfortable with parsing the travis config. It's too fragile, it could break if even with simple changes of the xml. Can't this be `-Dscala.version=$TRAVIS_SCALA_VERSION`